python/plugins/db_manager/db_plugins/plugin.py

```diff
@@ -1111,15 +1129,18 @@ def update(self, new_name, new_type_str=None, new_not_null=None, new_default_str
             new_not_null = None
         if self.default2String() == new_default_str:
             new_default_str = None
-
+        if self.comment == new_comment:
+            # Update also a new_comment
+            new_comment = None
```

Comment:
I'm not sure I get the logic here, is it to skip updating the comment if it has not changed?

Reply:
For me it is the idea. I preferred to use the same syntaxes as the others fields, so it keep a same logic.
It's in the case you update a fields name without changing the comment, it won't try to change the comment.